src/ducks/settings/TogglePane.styl

```diff
@@ -0,0 +1,5 @@
+.TogglePane__main
+    margin-bottom 3rem
```

Comment:
I don't understand why we need such a space. Maybe it  should be the job of an utility class to do that ? The spacing classes from Cozy-UI do not do the job ?